src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -5,23 +5,33 @@ import akka.http.scaladsl.server.Route
 import com.typesafe.scalalogging.LazyLogging
 import org.broadinstitute.dsde.workbench.leonardo.model.{ClusterName, GoogleProject}
 import org.broadinstitute.dsde.workbench.leonardo.service.ProxyService
+import org.broadinstitute.dsde.workbench.model.WorkbenchUserEmail
+
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Remove this import and put
```
implicit val executionContext: ExecutionContext
```
inside the trait. That way the EC is provided by the `LeoRoutes` subclass, and we don't have to import the global one here.